### PR TITLE
HiddenFromAddressLists and CutOver migration

### PR DESCRIPTION
**Pull request by diecknet**
Added info that mailboxes with HiddenFromAddressLists are not supported. Merged bullet point 4 and 5 in section "Prepare for a cutover migration", since they're both about the permissions.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/cutover-migration-to-office-365.md [hunk 1]
```diff
@@ -182,7 +183,7 @@ A migration endpoint contains the settings and credentials needed to connect the
 
 6. On the **Enter on-premises account credentials** page, enter information in the following boxes:
 
-   - **Email address**: Type the *email address* of any user in the on-premises Exchange organization that will be migrated. Microsoft 365 or Office 365 will test the connectivity to this user's mailbox.
+   - **Email address**: Type the *email address* of any user in the on-premises Exchange organization that will be migrated. Microsoft 365 or Office 365 will test the connectivity to this user's mailbox. Make sure that this mailbox is NOT hidden from the address lists.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/cutover-migration-to-office-365.md:186]:
```suggestion
   - **Email address**: Type the *email address* of any user in the on-premises Exchange organization that will be migrated. Microsoft 365 or Office 365 will test the connectivity to this user's mailbox. Make sure that this mailbox is _not_ hidden from the address lists.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**